ush/parsing_namelists_MOM6.sh

```diff
@@ -41,6 +41,10 @@ cat input.nml
 # ---------
 # Prepare local variables for use in MOM_input.IN from UFSWM
 # The ones already defined are left commented as a reminder
+# == MOM options to start from coarsed grained restarts, set to off by default
+local MOM6_INIT_FROM_Z=${MOM6_INIT_FROM_Z:-True}
+local MOM6_WARMSTART_FILE=${MOM6_WARMSTART_FILE:-"none"}
+local MOM6_INIT_UV=${MOM6_INIT_UV:-"zero"}
```

Comment:
NB: These variables are not in the MOM namelists for 0p25 or 5p00; only 0p50 and 1p00.

Reply:
correct. They are used with taking 0.25 MOM restarts and interpolating them to a lower resolution. Do you recommend them being at another location?

Reply:
Maybe just add a comment nothing they are only used for 0p50 and 1p00?

Reply:
completed